packages/react-instantsearch/src/core/InstantSearch.js

```diff
@@ -1,14 +1,14 @@
-import React, { PropTypes, Component, Children } from 'react';
+import React, {PropTypes, Component, Children} from 'react';
```

Comment:
I think you're not running the post-commit `prettier` hook, or with other settings.